tools/setup/postgres-init-dev-db

```diff
@@ -20,25 +45,8 @@ fi
 
 VAGRANTUSERNAME=$(whoami)
 
-if [[ $# == 0 ]]; then
-    USERNAME=zulip
-    PASSWORD=$("$(dirname "$0")/../../scripts/get-django-setting" LOCAL_DATABASE_PASSWORD)
-    DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
-    STATUS_FILE_NAME="migration_status_dev"
-elif [[ $# == 5 ]]; then
-    USERNAME=$1
-    PASSWORD=$2
-    DBNAME=$3
-    SEARCH_PATH=$4
-    STATUS_FILE_NAME=$5
-else
-    echo "Usage Instructions"
-    echo "Run with either no arguments (sets up devel db for local deploy--zulip with user zulip)"
-    echo "or specify <db-username> <password> <db-name> <user-schema-search-path> <migration-status-path>"
-    exit
-fi
-
+SEARCH_PATH=zulip,public
+PASSWORD=$("$(dirname "$0")/../../scripts/get-django-setting" LOCAL_DATABASE_PASSWORD)
```

Comment:
This should be above the `set -x`.

Reply:
I guess we haven't been careful about this before, and it's a password to the test database, so maybe it's fine.